docs/SupportedONNXOps-NNPA.md

```diff
@@ -12,32 +12,32 @@ Onnx-mlir currently supports ONNX operations targeting up to opset 18. Limitatio
-| **Gemm** |13 |- `alpha` and `beta` must be default value(1).<br>- Rank of `C` must be 1 or 2. If the rank is 1, the dimension of `C` must be the same with the seconde dimension of `B`. | |
-| **GlobalAveragePool** |1 |- Input shape must be 4D tensor(NCHW).<br>- Dimensions in `H` and `W` must be static. | |
-| **LSTM** |14 |- `direction` and `hidden_size` in `W` must have static dimensions.<br>- `R` must have static dimensions.<br>- `B` and `initial_h` have static dimensions if given. `B`'s direction dim must be 1 or 2.<br>- `P`(peepholes), `activation_alpha`, and `activation_beta` are not supported.<br>- `activations` must be `["Sigmoid", "Tanh", "Tanh"]`.<br>- `clip` is not supported.<br>- `input_forget` must be default value(0).<br>- `layout` is not supported. | |
-| **LeakyRelu** |16 |The operations immediately before and after the LeakyRelu operation must be executed on the NNPA. Otherwise, LeakyRelu is executed on the CPU. This limitation is set to avoid performance degradation. | |
-| **Log** |13 |Input tensor must have 4 dimensions. | |
-| **LogSoftmax** |13 | | |
-| **MatMul** |13 |Ranks of input tensors must be (Rank of A, Rank of B) = (M, N), where M >= 2 and N >= 2. | |
-| **Max** |13 |- Shape of input tensors must be the same since broadcasting is not supported.<br>- Input tensors must have static dimensions. | |
-| **MaxPool** |12 |- `auto_pad` must be `NOTSET`, `VALID`, and `SAME_UPPER`. If `NOTSET` is used, `pads` must be set so that the padding valid type or same upper.<br>- `ceil_mode` must be default value(0) <br>- Input and output tensors must be 4D tensors(N x C x H x W).<br>- `kernel_shape` must be static.<br>- `ceil_mode` must be default value(0).<br>- `dilations` must be default value(1). | |
-| **Min** |13 |- Shape of input tensors must be the same since broadcasting is not supported.<br>- Input tensors must have static dimensions. | |
-| **Mul** |14 |- Shape of input tensors should be the same since broadcasting is not supported.<br>- Input tensors must have static dimensions. | |
-| **Pow** |15 |- Exponent should be a scalar integer and less or equal to 64. | |
-| **ReduceMean** |18, 13 |- `keepdims` must be 1.<br>- Input tensor must be 4D tensors and `axis` must be [2, 3]. | |
-| **Relu** |14 |Input tensor must be less than or equal to 4 dimensions. | |
-| **Sigmoid** |13 |Input tensor must be less than or equal to 4 dimensions. | |
-| **Softmax** |13, 11 |- `axis` must be the last dimension, i.e. `rank - 1` or -1. | |
-| **Sub** |14 |- Shape of input tensors should be the same since broadcasting is not supported.<br>- Input tensors must have static dimensions. | |
-| **Sum** |13 |- All inputs must have the same static shape (Broadcasting not supported.)<br>- Single input not supported. | |
-| **Tanh** |13 |Input tensor must be less than or equal to 4 dimensions. | |
+| Op |Min Opset |Max Opset |Limitations |Notes |
```

Comment:
Nit: "Min Ver" and "Max Ver" instead of Min/Max Opset 

Basically I want to reduce the chance that someone looks at this list and thinks they have to convert their whole model to OpSet 14 because that's the "Max" (ie last updated) version of Add.

Sadly ONNX itself makes this confusing. The Op's version number is based on the OpSet where it was last updated.

Reply:
I am fine with the updates.... @AlexandreEichenberger what are your thoughts? This documented is generated when the `onnx-mlir/utils/documentOps.py` script gets run. So I will leave this to Alex :)

Reply:
> Nit: "Min Ver" and "Max Ver" instead of Min/Max Opset

@hamptonm1 would you mind changing this as part of your PR? Search for this line`header = ["Op", "Min Opset", "Max Opset", "Limitations"]`

Reply:
@AlexandreEichenberger Yeah I can change it.... I know where it is located